src-tauri/tauri.conf.json

```diff
@@ -8,7 +8,7 @@
     "withGlobalTauri": true
   },
   "package": {
-    "productName": "ChatGPT Next Web",
+    "productName": "BioChatter-Next",
     "version": "2.9.12"
```

Comment:
why is this version still here? it conflicts with the 1.0.0 we define in `package.json`, no?

Reply:
Yes, the version is still there, but we won't use this. do we need to delete this file?

Reply:
I don't know what the file does. Do you know?

Reply:
This file is used to build web app to a desktop app [Tauri](https://tauri.app/)

Reply:
So should we keep that? Does it work out of the box?

Reply:
We can just replace 2.9.12 with 1.0.0 and have bumpversion update this file as well. If the Tauri build works, that is.

Reply:
@slobentanzer , I introduced standard-version to bump version. It requires:
1. Always using squash commits when merging PRs
2. Add types to commit message like `feat:`, `fix:` `refactor:`, `test:`, and `doc:`

Then after each PR, we can run `yarn run release` to bump version and merge to main.

Reply:
I don't know that, but to be clear: we add the types to the squash message of the merge, right? Not to the individual ones in the PR itself.

Reply:
Yes, only need to add type to squash message of the PR. Let's try it to see if it works

Reply:
OK :)